src/rust/engine/fs/store/src/snapshot_ops.rs

```diff
@@ -234,11 +234,11 @@ pub trait SnapshotOps: Clone + Send + Sync + 'static {
       .map_err(|err| format!("Error matching globs against {directory_digest:?}: {}", err))?;
 
     let mut files = HashMap::new();
-    input_tree.walk(SymlinkBehavior::Aware, &mut |path, entry| match entry {
+    input_tree.walk(SymlinkBehavior::Oblivious, &mut |path, entry| match entry {
```

Comment:
Possibly worth mentioning the symlink obliviousness on `DigestSubset`? Or maybe giving it a boolean/enum to control it?